src/StepDefinition/MinkElementTrait.php

```diff
@@ -184,4 +184,22 @@ public function textAppearsTimesInElement($text, $element, $number, $selector =
     }
   }
 
+  /**
+   * Visit node page by title.
+   *
+   * @When I visit node with title :title
+   */
+  public function visitNodeByTitle($title) {
+    $entityTypeManager = \Drupal::entityTypeManager();
+    $nodes = $entityTypeManager
+        ->getStorage('node')
+        ->loadByProperties(['title' => $title]);
+    if ($node = reset($nodes)) {
+      $this->visitPath($node->toUrl()->getInternalPath());
```

Comment:
minor: why does it use the internal path? $node->toUrl() should be able to cast to (string) fine.